### PR TITLE
allow for multiple files to be discarded (or opened) at once

### PR DESCRIPTION
**Pull request by tnrich**
Fixes https://github.com/letmaik/vscode-git-tree-compare/issues/98

@letmaik plz review :)

### CODE REVIEW

**Review by letmaik — CHANGES_REQUESTED** [review]
Looks good, but this is breaking the single file case now. Can you check if both can be supported?

File: package.json [hunk 1]
```diff
@@ -147,6 +147,12 @@
                 "title": "View as Tree",
                 "icon": "$(list-tree)",
                 "category": "Git Tree Compare"
+            },
+            {
+                "command": "gitTreeCompare.discardSelectedChanges",
+                "title": "Discard Selected Changes",
+                "icon": "$(discard)",
+                "category": "Git Tree Compare"
```
[the remark this answers is not in the document]
Comment (tnrich) [reply package.json:155]: whoops yep looks left over

**Comment by tnrich** [comment]
@letmaik fixed

**Review by letmaik — APPROVED** [review]
All working fine, thanks for the PR! :)

**Pull request merged**